python/pylibcudf/pylibcudf/strings/split/split.pyx

```diff
@@ -27,7 +28,8 @@ __all__ = [
 ]
 
 cpdef Table split(
-    Column strings_column, Scalar delimiter, size_type maxsplit, Stream stream=None
+    Column strings_column, Scalar delimiter, size_type maxsplit, Stream stream=None,
+    DeviceMemoryResource mr=None,
```

Comment:
Nit: And adding the `mr`s in this file didn't lead to each argument being put on a new line (if it was done by AI)

Reply:
Yes, it was done by AI. I think because it didn't overflow line lengths here so it wasn't forced to, whereas in many other places it was? I can change it here too since I'm fixing your above comment too anyway.